test/index.test.js

```diff
@@ -1117,4 +1117,225 @@ describe('index', () => {
             });
         });
     });
+
+    describe('_addWebhook', () => {
+        const oauthToken = 'oauthToken';
+        const scmUri = 'bitbucket.org:arnoldWeber/wyattLib:mybranch';
```

Comment:
`{1234}` isn't a valid UUID. 

Changing this value, and all the other values introduced, wouldn't be different than how the other tests are incorrectly structured also. And if you'd like us to update the tests to use proper UUIDs, then we'd consider that maintenance creep and would prefer to defer that for another time.